adapters/dogstatsd/dogstatsd.go

```diff
@@ -32,8 +33,11 @@ type dogstatsdAdapter struct {
 }
 
 func (a *dogstatsdAdapter) Stream(logstream chan *router.Message) {
+	var logspout_dogstatsd_enabled = os.Getenv("LOGSPOUT_DOGSTATSD_ENABLED")
 	for m := range logstream {
-		a.inc(m)
+		if logspout_dogstatsd_enabled != "false" {
```

Comment:
I wouldn't check this for every logstream, since it's not like it can be set differently per logstream.  So maybe move this out a level?